policies_and_profiles_guide/_topics/chap_events.md

```diff
@@ -11,7 +11,6 @@ Events are triggers that cause a condition to be tested. Control provides severa
 | Orchestration Lifecycle   | Events related to orchestration lifecycle, such as retirement.                                                                                                                                       |
 | Physical Server Operation | Events related to the connection state of a physical server.                                                                                                                                         |
 | VM Configuration          | Events associated with a change in configuration of a virtual machine. These include, but are not limited to, clone, create, template create, and settings change.                                   |
-| VM Lifecycle              | Events such as virtual machine discovery, provisioning, and virtual machine retirement.                                                                                                              |
```

Comment:
This should not be removed.  We do still have the lifecycle events, we just don't have it as a dedicated model.